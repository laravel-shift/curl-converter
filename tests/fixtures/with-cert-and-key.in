curl --cert /path/to/cert --key /path/to/key https://example.com
