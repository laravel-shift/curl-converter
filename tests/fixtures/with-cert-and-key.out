Http::withOptions([
        'cert' => '/path/to/cert',
        'ssl_key' => '/path/to/key',
    ])
    ->get('https://example.com');
